cmd/format-coverage.go

```diff
@@ -32,7 +33,7 @@ type CoverageFormatter struct {
 var formatOptions = CoverageFormatter{}
 
 // a prioritized list of the formatters to use
-var formatterList = []string{"simplecov", "lcov", "coverage.py", "clover", "gocov", "gcov", "cobertura"}
+var formatterList = []string{"simplecov", "lcov", "coverage.py", "clover", "gocov", "gcov", "cobertura", "jacoco"}
```

Comment:
We could, but that is a prioritized look up list. Sorting it alphabetically would change how we look up for matters. That was a requirement early on. If it's no longer a requirement we can delete the list altogether. :)